composer.json

```diff
@@ -39,20 +38,12 @@
       "dealerdirect/phpcodesniffer-composer-installer": true,
       "composer/installers": true,
       "phpstan/extension-installer": true,
-      "cweagans/composer-patches": true
+      "cweagans/composer-patches": false
```

Comment:
Shouldn't we remove this entry instead of setting it to `false`, given we're no longer using the package?

Reply:
The plugin could still be used by one of the other dependencies. Explicitly setting this to false prevents it from running unexpectedly. So this is good from a security point of view.